language-support/ts/codegen/tests/ts/build-and-lint-test/src/__tests__/test.ts

```diff
@@ -508,3 +509,24 @@ test('stream close behaviour', async () => {
     'close',
   ]);
 });
+
+test('party API', async () => {
+  const p = (id: string): PartyInfo => ({identifier: id, displayName: id, isLocal: true});
+  const ledger = new Ledger({token: ALICE_TOKEN, httpBaseUrl: httpBaseUrl()});
+  const parties = await ledger.fetchPartiesInfo([ALICE_PARTY, "unknown"]);
+  expect(parties).toEqual([p("Alice"), null]);
+  const rev = await ledger.fetchPartiesInfo(["unknown", ALICE_PARTY]);
+  expect(rev).toEqual([null, p("Alice")]);
+
+  const allParties = await ledger.fetchAllPartiesInfo();
+  expect(_.sortBy(allParties, [(p: PartyInfo) => p.identifier])).toEqual([p("Alice"), p("Bob")]);
```

Comment:
Here `lodash` buys us a tiny bit but `.sort` accepts a custom comparator so only very very little imho.

Reply:
`.sort` is in-place mutation, so I tend to avoid it by default. There's no specific reason to stay functional here, but there's also no strong perf requirement to mutate in place. Happy to change if you want.

Reply:
Ah right, I had forgotten about that. Then let’s keep the immutable version. Definitely not worried about performance, I just like to stick to the stdlib when it doesn’t make a difference but immutability seems good enough.

Reply:
It's not uncommon in the JS world to pretend `lodash` is part of the stdlib.